main.cpp

```diff
@@ -308,6 +319,7 @@ struct WaveGenerator
     double modulator = 24.5;
     std::string title = "Wave Generator 1";
     double filter = 35.75;
+    double mSineWaveFreq = 440;
```

Comment:
don't use `m` in member variable names
If a variable is used in a function, and it is not declared in that function or part of the function parameter list, it is obvious that the variable is a member variable. 
- [ ] revise